src/server/enrichment/validation/index.js

```diff
@@ -5,100 +5,144 @@ const { validTargetDb } = require('./validity-info');
 const qs = require('query-string');
 const { cleanUpEntrez } = require('../helper');
 
-const defaultOptions = {
-  'output': 'mini',
-  'organism': 'hsapiens',
-  'target': 'HGNC',
-  'prefix': 'ENTREZGENE_ACC'
-};
+const GCONVERT_URL = 'https://biit.cs.ut.ee/gprofiler_archive3/r1741_e90_eg37/web/gconvert.cgi';
+const FETCH_TIMEOUT = 5000; //ms
 
-class InvalidInfoError extends Error {
-  constructor(invalidOrganism, invalidTargetDb, message) {
-    super(message);
-    this.invalidTargetDb = invalidTargetDb;
-    this.invalidOrganism = invalidOrganism;
+class InvalidParameterError extends Error {
```

Comment:
We might not need an invalid parameter error here.  We could just use a normal error